Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -547,7 +558,8 @@ bool BluetoothReal::OpenDevice(libusb_device* device)
   const int ret = libusb_open(m_device, &m_handle);
   if (ret != 0)
   {
-    PanicAlertT("Failed to open Bluetooth device: %s", libusb_error_name(ret));
+    m_last_open_error = StringFromFormat(
```

Comment:
I know @lioncash did some work to switch over some uses of `StringFromFormat` to `fmt`, would that be worthwhile to do where you're using `StringFromFormat` here and below?

Reply:
Good idea! I've replaced both uses of StringFromFormat with fmt.